config/sample-config.yaml

```diff
@@ -812,3 +818,6 @@ frontend:
   alertNamespaces:
     - netobserv
   sampling: 50
+  deduper:
+    mark: false
+    merge: true
```

Comment:
isn't better to stay with the current default till we change it ?

Reply:
Sure !